swagger-templates/src/main/resources/OktaJava/licenseInfo.mustache

```diff
@@ -1,5 +1,5 @@
 {{!
-    Copyright 2017 Okta
+    Copyright 2020-Present Okta, Inc.
```

Comment:
Keep the original date. The copyright start date is usually the most important one
```suggestion
    Copyright 2017-Present Okta, Inc.
```